src/errors.ts

```diff
@@ -0,0 +1,41 @@
+import { Request, Response, NextFunction } from 'express';
+
+export class NotFoundError extends Error {
+    public status = 404;
```

Comment:
no big deal, but I would name it statusCode vs status. How about using https://www.npmjs.com/package/http-status-codes ?

Reply:
I'm not sure it's relevant to add a full library for only 1 status code, will change status to statusCode

Reply:
nah, you are right that it's a bit far-fetched to add a lib for this at this point. However, if you have time I would at least create a static map or constants so that we avoid magic-not-so-magic numbers. for instance,

`export const HTTP_STATUS_CODE_NOT_FOUND = 400;`

but only if you have time to do it and think it's worth it.

Reply:
I add 2 const in errors file. Tell me if it's fine for you or you want an enum ?